packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -16,21 +16,31 @@ contract SystemConfig is OwnableUpgradeable, Semver {
     uint256 public overhead;
     uint256 public scalar;
     bytes32 public batcherHash;
+    uint64 public gasLimit;
+
+    /**
+     * @notice Minimum gas limit. This should not be lower than the maximum deposit
+     *         gas resource limit in the ResourceMetering contract used by OptimismPortal,
+     *         to ensure the L2 block always has sufficient gas to process deposits.
+     */
+    uint64 public constant MINIMUM_GAS_LIMIT = 8_000_000;
```

Comment:
How easy is it to tie this value to the resource metering contract value? It's good to have a check in this contract rather than not messing it up during deploys, but this can get out of sync with the other contract.

Reply:
We would have to tightly couple the contracts, which complicates set up. It's a constant, so maybe we add a test that the minimum gas limit is always equal or larger than the maximum L1 resource meter gas in the portal? I'll add it to the solidity testing issue.